src/core/currency/wallet/currency-wallet-pixie.js

```diff
@@ -144,12 +144,15 @@ export const walletPixie: TamePixie<CurrencyWalletProps> = combinePixies({
         payload: { height, walletId }
       })
       if (engine.getStakingStatus != null) {
-        engine.getStakingStatus().then(stakingStatus => {
-          input.props.dispatch({
-            type: 'CURRENCY_ENGINE_CHANGED_STAKING',
-            payload: { stakingStatus, walletId }
+        engine
+          .getStakingStatus()
+          .then(stakingStatus => {
+            input.props.dispatch({
+              type: 'CURRENCY_ENGINE_CHANGED_STAKING',
+              payload: { stakingStatus, walletId }
+            })
           })
-        })
+          .catch(error => input.props.onError(error))
```

Comment:
I think the intention was to load the staking balances asynchronously, letting the wallet appear in the list even if we don't know the balance. Hence, sending the error to the log instead of blocking the load. If they get 0 for their staked funds, that would be analogous to having the wallet fail to sync in other ways.